src/spring/azext_spring/_params.py

```diff
@@ -864,6 +865,11 @@ def prepare_logs_argument(c):
     for scope in ['create', 'update']:
         with self.argument_context('spring application-configuration-service {}'.format(scope)) as c:
             c.argument('generation', arg_type=get_enum_type(ConfigurationServiceGeneration), help='Generation of Application Configuration Service.')
+            c.argument('refresh_interval', type=int,
+                       validator=validate_refresh_interval,
+                       help='Number of the repository refresh interval (in seconds). '
```

Comment:
'Specify the interval (in seconds) for refreshing the repository. Use 0 to turn off automatic refresh. An interval of at least 60 seconds is recommended.'